src/measures/MeanQuadraticDistance.jl

```diff
@@ -0,0 +1,46 @@
+#X"Use for calculating sqr distance" 
+
+#first we need a new structur 
+
+struct MQD end 
+
+#then compute the function that takes imput from object sheep::Sheeps 
+function SqrDist(sheep, sheep1, gridsizestorage) 
```

Comment:
```suggestion
function sqr_dist(sheep, sheep1, gridsizestorage) 
```
in julia functions are usually lower case. You could also consider naming it more expressive like e.g.: `squared_distance`.
You need to also change the name where the function is used.